src/main/java/com/revature/models/Wishlists.java

```diff
@@ -0,0 +1,23 @@
+package com.revature.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Entity
+@NoArgsConstructor
+@AllArgsConstructor
+public class Wishlists {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+    private int user_id;
```

Comment:
Use a User object here instead of the user_id and define the @ManyToOne relationship like in the Order class.